### ISSUE TITLE
Remove all models that are not used on the system.

### ISSUE THREAD

**Issue by nherriot** [nherriot/noisy-atom-portal#45]
There are old models that are not used on the system. The only application that we use is the **blog** application.

Compare a vanilla Django application (e.g. a hello world django server) with noisy_atom. Make sure **ALL** models are removed.

After that wipe all migrations that are on the system. Then generate a brand new migration script. There should only be one migration script.
We will use this to wipe our live Database, since it has no data, and start everything from fresh. To complete the task:

- [x] Compare models with vanilla django server and remove all models and apps that are not in the vanilla list except the **blogs** app.
- [x] Remove all migrations. Check with documentation that this will not break anything.
- [x] Do **one** single migration. Check it works.
- [x] Verify django admin works.
- [x] Verify you can add/remove/update a blog.
- [ ]  Verify that all applications that are not used for django are removed. Remove all DJANGO applications from the syste.

**Comment by DilshadGit** [comment]
1. In the settings.py change MIDDLEWARE_CLASSES  to MIDDLEWARE  to get access to other pages like admin or others, the _CLASSES has removed in Django==2.

**Comment by DilshadGit** [comment]
2. Removed all app models include except Blog runserver working No bugs.
3. All models has been deleted delete the migrations run makemigration and migrate. All migrations folder has one migrations all working no error and no bugs so fare. Also create new user run python manage.py createsuperuser work fine.
4. The admin page working can login and logout.

**Comment by DilshadGit** [comment]
- [ ]  Verify you can add/remove/update a blog.
The create, delete both working but update not change the content or if you change the image not able to make any update the changes has been made.
- blogs/templates/update.html
``` <form action="" method="POST" accept-charset="utf-8"> ```
Because the form update is include media need to add following:
```<form action="" method="POST" accept-charset="utf-8" enctype="multipart/form-data">```

- blogs/views.py
The  ```form = PostForm(request.POST, request.FILES, instance=blog_post)``` was identified after the post because we update the form object before if statement. For adding new post need create after if request.method == 'POST' , but if we want to update the post already created and save in the database we have to create the form object before the if request.method == 'POST' because use as GET and POST changed to:
```form = PostForm(request.POST or None, request.FILES or None, instance=blog_post)```
(or None) has added to update the request or nothing the same for the request.FILES.

We don't need form object and context after else has to return only message if there is an error.
```
 context = {'form': form}
            return render(request, template_name, context)
```
Last change removed form object:
``` form = PostForm(instance=blog_post) ``

The context here is static not updated need also to change to instance object.
```
context = {
        'main_title': 'Blogs At Noisy Atom',
        'description1': 'Read all about the latest thought, what we are working on and how we are progressing! A brain dump of where we are at!',
        'description2': 'We hope you enjoy our thoughts...',
        'description3': '',
        'form': form}
```
Changed to:
```context = {
        'main_title': blog_post.title,
        'blog_post': blog_post,
        'form': form
    }
```

**Comment by nh-accedian** [comment]
> context = {'form': form}
>  return render(request, template_name, context)

Please submit in a pull request.

**Comment by nh-accedian** [comment]
> 2. Removed all app models include except Blog runserver working No bugs.
> 3. All models has been deleted delete the migrations run makemigration and migrate. All migrations folder has one migrations all working no error and no bugs so fare. Also create new user run python manage.py createsuperuser work fine.
> 4. The admin page working can login and logout.

Please submit in pull request.

**Comment by nh-accedian** [comment]
> * [x]   Verify you can add/remove/update a blog.
>   The create, delete both working but update not change the content or if you change the image not able to make any update the changes has been made.
> * blogs/templates/update.html
>   `<form action="" method="POST" accept-charset="utf-8">`
>   Because the form update is include media need to add following:
>   `<form action="" method="POST" accept-charset="utf-8" enctype="multipart/form-data">`
> * blogs/views.py
>   The  `form = PostForm(request.POST, request.FILES, instance=blog_post)` was identified after the post because we update the form object before if statement. For adding new post need create after if request.method == 'POST' , but if we want to update the post already created and save in the database we have to create the form object before the if request.method == 'POST' because use as GET and POST changed to:
>   `form = PostForm(request.POST or None, request.FILES or None, instance=blog_post)`
>   (or None) has added to update the request or nothing the same for the request.FILES.
> 
> We don't need form object and context after else has to return only message if there is an error.
> 
> ```
>  context = {'form': form}
>  return render(request, template_name, context)
> ```
> 
> Last change removed form object: `form = PostForm(instance=blog_post)`
> 
> The context here is static not updated need also to change to instance object.
> 
> ```
> context = {
>         'main_title': 'Blogs At Noisy Atom',
>         'description1': 'Read all about the latest thought, what we are working on and how we are progressing! A brain dump of where we are at!',
>         'description2': 'We hope you enjoy our thoughts...',
>         'description3': '',
>         'form': form}
> ```
> 
> Changed to:
> 
> ```
> context = {
>         'main_title': blog_post.title,
>         'blog_post': blog_post,
>         'form': form
>     }
> ```

Please submit in pull request.

**Comment by nherriot** [comment]
This is closed with pull request #56

**Comment by nherriot** [comment]
This is not closed with pull request. There are a few changes required.
The issue does not state that the django applications need to be deleted. I have updated the description to include the deletion of the applications.
